.ci/deploy-linux.sh

```diff
@@ -33,7 +33,12 @@ if [ "$DEPLOY_APPIMAGE" = "true" ]; then
     printf "#include <bits/stdc++.h>\nint main(){std::make_exception_ptr(0);std::pmr::get_default_resource();}" | $CXX -x c++ -std=c++2a -o ./appdir/usr/optional/checker -
 
     # Package it up and send it off
-    ./squashfs-root/usr/bin/appimagetool /rpcs3/build/appdir
+    if [ -n "$CIRRUS_CI" ]; then
+        ./squashfs-root/usr/bin/appimagetool ./appdir
+    else
+        ./squashfs-root/usr/bin/appimagetool /rpcs3/build/appdir
+    fi
+    
```

Comment:
It may be worth considering changing a lot of the logic in this file. Having too many ifs always feels odd, and it's worth asking if its worth keeping them. I know neko said maybe we should split the responsibilities, but I think it would be worth unifying them.

cc @hcorion since he's dealt with this for so long too

Reply:
Yep I agree, there should definitely be some cleanup here. The ifs get messy and will undoubtedly grow when we switch to whatever new CI of the week. These should either come from an env variable (set as a build env var so the scripts remain the same) or if we could unify cirrus into working with a /rpcs3/ directory that would work too